docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -138,6 +137,409 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=include-segment-file-sizes]
 [[cluster-nodes-stats-api-response-body]]
 ==== {api-response-body-title}
 
+[[cluster-nodes-stats-api-response-body-indices]]
+===== `indices` section
+
+The `indices` flag can be set 
+to retrieve information about indices:
```

Comment:
is this correct ?  `GET _nodes/stats` does not accept a `indices` param and this info is returned by default.  (or do i have the wrong API ?_

Reply:
The endpoint would be `_nodes/stats/indices`.

I borrowed the wording here from https://github.com/elastic/elasticsearch/pull/47525/files#diff-055d8b69ee66d0876311d63bc11fb382L144, but I can either remove it throughout. I feel like we explain how to pull particular metric groups well in the rest of the page.

Reply:
Removed this and similar sentences with 9b5c837.

Reply:
I don't have a strong opinion to include or not, I was thrown off by the term `flag` as I mentally mapped that to a `?query_param`.

Reply:
Thanks! I think it works better without anyway.